.github/workflows/test-tutor-aspects.yml

```diff
@@ -0,0 +1,43 @@
+name: Test Tutor Aspects
+
+on:
+  push:
+    branches: [ main ]
+  issue_comment:                                     
+    types: [created, edited, deleted]
+  pull_request:
```

Comment:
I think we only need this on PRs to main _or_ pushes to main, but not both? If we add it to PRs we'll need to wait for it to finish before merging, which may not be the worst. 🤷